pilot/pkg/bootstrap/server.go

```diff
@@ -145,6 +150,7 @@ type Server struct {
 	GRPCListeningAddr       net.Addr
 	SecureGRPCListeningAddr net.Addr
 	MonitorListeningAddr    net.Addr
+	MCPClientListeningAddr  net.Addr
```

Comment:
Comments ( for all things you add), what is this ?

Reply:
We'll remove that.